### DISCUSSION TITLE
Lesson 189 - RuntimeError / Load EfficientNet_B0 weights

### DISCUSSION THREAD

**Discussion by progr21** [mrdbourke/pytorch-deep-learning#719] [Q&A]
Hello Team,

I faced the RuntimeError depicted in the attatched picture when trying to load the pretrained weights EfficientNet_B0 model. Could you please assist with this issue?

Thank you in advance!!

![Στιγμιότυπο οθόνης 2023-11-12 211718](https://github.com/mrdbourke/pytorch-deep-learning/assets/93767059/d269a2dd-5e9d-425d-91d4-e82f44ebd3e3)

**Comment by mrdbourke** [comment]
Hi @progr21 , 

Looks like this is a known issue with `torchvision`, there's a quick fix is this issue thread here: https://github.com/mrdbourke/pytorch-deep-learning/issues/696 

It should be fixed in a future version of `torchvision`. 

Best, 

Daniel
